### DISCUSSION TITLE
Abstimmungen und Wahlen im Namen einer Organisation

### DISCUSSION THREAD

**Discussion by DerUtz** [OpenSlides/OpenSlides#6661] [General]
Ist es möglich im Namen einer Organsiation als Delegierte:r abzustimmen? Das Stimmrecht müsste dann auf die Organisationen beschränkt sein und die Delegierten machen dies in deren Auftrag. Stimmrechtsübertragung ist bisher nicht möglich, da nur an eine:n Delegierte:n übertragen werden kann. Hier müsste das Stimmrecht an mehrere Delegierte:n übertragen werden können.

**Comment by jsangmeister** [comment]
Delegating the vote to multiple people is already discussed in https://github.com/OpenSlides/openslides-backend/issues/1883. If you then create a user with the name of your organization, this should fulfil your need. It is unfortunately not clear when and if the feature will be implemented.

**Reply by DerUtz** [reply]
will this also be available with SAML?

**Reply by jsangmeister** [reply]
Yes, these features are independent of each other. Or do you mean if you can set the vote delegations via your identity provider?

**Reply by DerUtz** [reply]
It woult be nice to set this with identity provider

**Reply by jsangmeister** [reply]
This is potentially planned, but still a long way down the road. The first step will be to manage group memberships via the IDP.
